app/models/location.rb

```diff
@@ -40,36 +40,36 @@ class Location < ApplicationRecord
 
   geocoded_by :address
 
-  attr_accessor :country
-
   def address
-    [address_line1, zip_code, city, state,
-     ISO3166::Country.find_country_by_alpha2(country_code)].compact.join(', ')
+    [
+      address_line1, zip_code, city, state,
+      ISO3166::Country.find_country_by_alpha2(country_code)&.unofficial_names&.first
```

Comment:
`ISO3166::Country.find_country_by_alpha2(country_code)&.unofficial_names&.first`

replace this by `country` directly?

Reply:
Oh, you mean the model method country. Haha, yes, good idea!